core-web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dotcms-ui",
-  "version": "23.3.0-next.16",
```

Comment:
please update these values accordingly. It shouldn't be merged into master

Reply:
This one is fine... matches master, master is bad tho.

https://github.com/dotCMS/core/blob/master/core-web/package.json#L3